src/views/virtualmachines/details/tabs/details/components/sections/UserList/hooks/useActiveUsersColumnsVm.ts

```diff
@@ -0,0 +1,46 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  K8sResourceCommon,
+  TableColumn,
+  useActiveColumns,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { sortable } from '@patternfly/react-table';
+
+const useActiveUsersColumnsVm = (): { title: string; id: string }[] => {
+  const { t } = useKubevirtTranslation();
+
+  const columns: TableColumn<K8sResourceCommon>[] = [
```

Comment:
here [1] you are using on VirtualizedTable with V1VirtualMachineInstanceGuestOSUser type so
this should be `const columns: TableColumn<V1VirtualMachineInstanceGuestOSUser>[] = ....`